xblockutils/public/studio_edit.js

```diff
@@ -65,6 +66,11 @@ function StudioEditableXBlockMixin(runtime, element) {
                 }
             });
         }
+
+        if (type == 'datepicker' && datepickerAvailable) {
+            $field.datepicker('destroy');
+            $field.datepicker({dateFormat: "m/d/yy"});
```

Comment:
@tusbar I'm afraid I'm not very familiar with how localization works in edX instances, especially when it comes to XBlocks. Are there any docs that I can read?

Just to make sure, some format is needed as otherwise it kind of works, but JS and python formats are quite different, so it just jumps back and forth between formats + python format is not self-consistent - if datetime field is not modified saving XBlock fails.